test/entrypoint/srv_ddl.lua

```diff
@@ -166,8 +173,8 @@ local ok, err = errors.pcall('CartridgeCfgError', cartridge.cfg, {
         'customers-storage',
         'cartridge.roles.crud-router',
         'cartridge.roles.crud-storage',
-    },
-})
+    }}, { readahead = 20 * 1024 * 1024 }
```

Comment:
Is it fair to do performance tests with non-default readahead?

Reply:
I don't mind keeping it as is, but, please, add a comment as guidance for future developers: why this value is suitable, but default is not.

Reply:
(Reminder.)

Reply:
Resolved now.